Framework/DataHandling/src/LoadMask.cpp

```diff
@@ -504,344 +675,55 @@ void LoadMask::parseXML() {
     } else if (pNode->nodeName().compare("ids") == 0) {
       // Node "ids"
       if (ingroup) {
-        this->parseSpectrumNos(value, m_MaskSpecID);
+        parseRangeText(value, singleSp, pairSp);
+        // this->parseSpectrumNos(value, m_maskSpecID);
```

Comment:
I wanted to remove this code and discussed removal of this code with Nick and he suggested me to keep it. Do not know why, but may be he has some ideas for it.